engo_glfw.go

```diff
@@ -1,4 +1,4 @@
-//+build !netgo,!android
+// +build darwin,!arm,!arm64 linux windows
```

Comment:
Maybe you would need a line `// +build !ios` for iOS emulators (where build tags are `darwin` & `amd64` & `ios`).
Additionally, you need `// +build !android`.